common/src/main/java/com/wynntils/utils/Utils.java

```diff
@@ -4,24 +4,8 @@
  */
 package com.wynntils.utils;
 
-import com.wynntils.core.WynntilsMod;
-import com.wynntils.wc.Models;
-import net.minecraftforge.eventbus.api.IEventBus;
-
 /**
  * This is a "high-quality misc" class. Helper methods that are commonly used throughout the project
  * without an aspect on minecraft can be put here. Keep the names short, but distinct.
  */
-public class Utils {
```

Comment:
@Incompleteusern You did not think we should have a generic Util class for a few select methods?

Reply:
The thing is both of those methods have better places to be, or in the case of the first one is unnecessary